pallets/auctions/src/lib.rs

```diff
@@ -91,25 +74,30 @@ pub mod pallet {
 		// This type is needed to convert from Currency to Balance
 		type CurrencyBalance: From<Self::Balance>
 			+ Into<<<Self as crate::Config>::Currency as Currency<<Self as frame_system::Config>::AccountId>>::Balance>;
+
+		/// Limit of auction name length
+		type AuctionsStringLimit: Get<u32>;
```

Comment:
add `#[pallet::constant]` attribute

This applies to all constants

Reply:
yeah I was wondering why this was missing, thx.